family_foto/__init__.py

```diff
@@ -82,17 +83,17 @@ def create_app(test_config: dict[str, Any] = None, test_instance_path: str = Non
 
     if os.path.exists(app.instance_path + '/app.db'):
         Migrate(app, db)
-    else:
-        db.init_app(app)
-        db.app = app
-        db.create_all()
+    db.init_app(app)
+    db.app = app
+    db.create_all()
 
     login_manager.init_app(app)
 
     from family_foto.web import photos, videos
     flask_uploads.configure_uploads(app, (photos, videos))
 
-    add_roles()
+    with app.app_context():
```

Comment:
what does this do?

Reply:
It pushes the application into the call. This makes sure we can access the database. It should work without, but it was recommended here:
https://flask.palletsprojects.com/en/1.1.x/appcontext/#manually-push-a-context

And as this error occurred I added it.